docs/gcc-Tips.rst

```diff
@@ -32,27 +32,34 @@ Some explanatory pictures:
 .. image:: ./_static/images/gcc-table.png
    :align: center
    :target: ./_static/images/gcc-table.png
+   :alt: picture comparing depth with number of threads with each implementation where red color means bad combination and greener areas mean good combination
 
 .. image:: ./_static/images/gcc-bars.png
    :align: center
    :target: ./_static/images/gcc-bars.png
+   :alt: picture of a simple bar chart against running time
 
 .. image:: ./_static/images/gcc-chart.png
    :align: center
    :target: ./_static/images/gcc-chart.png
+   :alt: this picture is a rendition of first where y-axis is cost and x-axis is number of threads comparing the four algorithms
 
 .. image:: ./_static/images/gcc-comparison-1.png
    :align: center
    :target: ./_static/images/gcc-comparison-1.png
+   :alt: a bar chart comapring Light G.B.M performance versus compilation flags which is maximum at depth 8 with Light G.B.M version 2, flag O 3
 
 .. image:: ./_static/images/gcc-comparison-2.png
    :align: center
    :target: ./_static/images/gcc-comparison-2.png
+   :alt: a bar chart comapring Light G.B.M cumulative performance versus compilation flags which is maximum at depth 8 with Light G.B.M version 2, flag O 3
```

Comment:
```suggestion
   :alt: a set of 4 vertical bar charts comparing Light G B M performance versus compilation flags. For most settings of max depth, best performance was achieved with flags dash O 3, dash M T U N E equals native.
```

I think this could have the same description as the image above it, just with "horizontal" switched to "vertical".